GovUk.Frontend.AspNetCore.Extensions/wwwroot/govuk/govuk-validation.js

```diff
@@ -32,8 +32,15 @@ function createGovUkValidator() {
     /**
      * Ensures the page title is prefixed with 'Error: ' when there is at least one .govuk-error-message displayed
      */
-    updateTitle: function () {
-      const prefix = "Error: ";
+      updateTitle: function () {
+        const getPrefix = document.querySelector(".govuk-error-prefix");
```

Comment:
You can simplify this with `document.querySelector("[data-govuk-error-prefix]")`. This will find the first element with that attribute, so you don't need to put a class on the element in order to find it.